### ISSUE TITLE
Demographics vs connections data consistency

### ISSUE THREAD

**Issue by MarcLoupias** [StateOfJS/StateOfJS#138]
Hello,

thanks for this great survey, i was waiting it for weeks !

But i have a big question regarding data consistency in [the connections page](https://2018.stateofjs.com/connections/).

How it is possible to survey `20K` developers and to get `92 681` React users displayed in the chart ?

I probably don't understand something but i can't figure out what is it.

**Comment by SachaG** [comment]
I agree it's a little weird, but it's the number of connections, not the number of developers. Maybe @plouc can explain better.

**Comment by MarcLoupias** [comment]
Ok i get it, it's not `92 681` React users but `92 681` answers for React + something else.

**Comment by MarcLoupias** [comment]
This is still super weird to me. If i took the React and ES6 connections in the graph, how it is possible to get a so small portion of React Users using ES6 ?

And more, how it is possible to not get 100% of TypeScript users using ES6 ? If you are using TypeScript you are using ES6, it is a superset.

Same on angular which is bundled with TypeScript by default, ...

This graph is confusing.

**Comment by SachaG** [comment]
Don't forget the data comes from people's answers, it's not going to be perfect.
